index.html

```diff
@@ -1 +1,11 @@
 Strona projektu
+
+<header>
+   <nav>
+     <ul>
+       <li><a href="index.html">Strona główna</a></li>
```

Comment:
I would suggest changing it to:

```suggestion
       <li><a href="index.html">Strona główna</a></li>
       <li><a href="about.html" target="_blank">O nas</a></li>
       <li><a href="contact.html">Kontakt</a></li>
```